src/components/Home.js

```diff
@@ -0,0 +1,70 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { BsArrowRightCircle } from 'react-icons/bs';
+import { NavLink } from 'react-router-dom';
+import { cryptoThunk, searchCrypto } from '../redux/crypto/cryptoReducer';
+import SearchBar from './Search';
+
+const Home = () => {
+  const crypto = useSelector((state) => state.crypto);
+  const dispatch = useDispatch();
+
+  const crypt = crypto.crypto;
+
+  useEffect(() => {
+    dispatch(cryptoThunk());
+  }, []);
+
+  const [value, setValue] = useState({ search: '' });
+
+  const handleSearch = (e) => {
+    e.preventDefault();
+    if (value.search === '') {
+      return;
+    }
+
+    dispatch(searchCrypto(value.search));
```

Comment:
- [ ] [OPTIONAL] I think it will be better to use a simple filter for the already fetched items on the homepage instead of calling a dispatch action